### ISSUE TITLE
should survey 123 connect not show the surveys I have in my organization already?

### ISSUE THREAD

**Issue by mhogeweg** [Esri/Survey123Community#48]
installed survey 123 connect 1.0.59 and signed in. then none of my existing surveys already published to ArcGIS Online are visible. is this by design?

**Comment by survey123** [comment]
Yes, this is the current design of Survey123 Connect. Survey123 Connect will only work with surveys that are saved locally to your machine. You can download the Code Sample for your survey from ArcGIS Online and extract it to your ~/ArcGIS/My Survey Designs folder to continue working with it in Survey123.

When the web interface for creating surveys online is ready, this piece of the puzzle should make working with existing surveys much easier. If there is still an obvious need for the desktop app to download survey projects after the web interface has been introduced, we will revisit this issue.
